### ISSUE TITLE
Unexpected error "Cannot read property 'statements' of null" occured

### ISSUE THREAD

**Issue by vvakame** [palantir/tslint#121]
Unexpected error "Cannot read property 'statements' of null" occured

How to reproduction.

```
$ node -v
v0.10.26
$ tslint -v
0.4.7
$ cat tslint.json
{
  "rules": {
    "use-strict": [true,
        "check-module",
        "check-function"
    ]
  }
}
$ cat test.ts
declare function foo():void;
$ wc test.ts
       1       3      29 test.ts
$ tslint -c tslint.json -f test.ts

/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/build/rules/useStrictRule.js:42
                this.checkUseStrict(node, node.block.statements);
                                                    ^
TypeError: Cannot read property 'statements' of null
    at UseStrictWalker.visitFunctionDeclaration (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/build/rules/useStrictRule.js:42:53)
    at FunctionDeclarationSyntax.accept (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:10581:28)
    at UseStrictWalker.SyntaxWalker.visitNode (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:20984:18)
    at UseStrictWalker.ScopeAwareRuleWalker.visitNode (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:67977:40)
    at UseStrictWalker.SyntaxWalker.visitNodeOrToken (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:20991:22)
    at UseStrictWalker.SyntaxWalker.visitList (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:21021:22)
    at UseStrictWalker.SyntaxWalker.visitSourceUnit (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:21033:18)
    at SourceUnitSyntax.accept (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:9749:28)
    at Rule.AbstractRule.applyWithWalker (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:67937:28)
    at Rule.apply (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/build/rules/useStrictRule.js:13:21)
    at Linter.lint (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:68064:45)
    at Object.<anonymous> (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint-cli.js:68198:25)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/vvakame/.nodebrew/node/v0.10.26/lib/node_modules/tslint/bin/tslint:3:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
```

Cheers!
